src/morphocut/annotation.py

```diff
@@ -63,26 +100,26 @@ def transform_stream(self, stream):
             try:
                 parent_img = self.prepare_input(parent, ("output"))
             except KeyError:
-                parent_img = img_as_ubyte(
-                    self.image, force_copy=True)
+                parent_img = skimage.color.gray2rgb(img_as_ubyte(
+                    self.image, force_copy=True))
                 self.prepare_output(parent, parent_img)
 
-            mask = self.prepare_input(obj, ("mask"))
+            #mask = self.prepare_input(obj, ("mask"))
 
             # Calculate radius of the dilation
-            area = np.sum(mask)
+            area = np.sum(self.mask)
             radius = self.dilate_rel * np.sqrt(area) + self.dilate_abs
 
             # Dilate mask
             dilated_mask = binary_dilation(
-                mask,
+                self.mask,
                 disk(radius))
 
             # Draw contours
             boundary_mask = skimage.segmentation.find_boundaries(dilated_mask, mode="outer")
 
-            color = (0, 255, 0)
+            #color = (0, 255, 0)
 
-            parent_img[self.parent_slice][boundary_mask] = color
+            parent_img[self.parent_slice][boundary_mask] = self.color
 
-            yield self.prepare_output(obj)
+            yield self.prepare_output(parent_img)
```

Comment:
This does not work as expected. This way, for each child object, we get the parent image with only this object marked.

Instead, we need to draw on `parent_image_out` that will be passed as reference.